src/Http/Controllers/PostController.php

```diff
@@ -5,13 +5,19 @@
 namespace DrewRoberts\Blog\Http\Controllers;
 
 use Illuminate\Http\Request;
+use DrewRoberts\Blog\Models\Topic;
+use DrewRoberts\Blog\Models\Series;
 use DrewRoberts\Blog\Models\Post;
 use Tipoff\Support\Http\Controllers\BaseController;
 
 class PostController extends BaseController
 {
-    public function __invoke(Request $request, Post $post)
+    public function __invoke(Request $request, Topic $topic, Series $series, Post $post)
```

Comment:
I don't know the data structures, but there may also be need for validation here if posts belong to series and series belong to topics.